Documentation/docs/appendix.md

```diff
@@ -225,9 +225,55 @@ If the SIMPLE_EXTRUSION is used, bottom topography can be optionally added to th
 		eqn = f(x,y) = some function of (x,y) as an equation
 	\end{TOPOGRAPHY}
 
-
-
-
-
-
-
+## Appendix E: ABAQUS mesh file format<a name="ABAQUS"></a>
+The ABAQUS mesh file which should use the extension `.inp` is divided into two parts. The top portion is the standard [Abaqus file format](https://abaqus-docs.mit.edu/2017/English/SIMACAEMODRefMap/simamod-c-model.htm). The skeleton of this form for a quadrilateral mesh with 122 corner nodes and 103 elements is as follows:
+
+	*Heading
+ 	 File created by HOHQMesh
+	*NODE
+	1, x1, y1, z1
+	2, x2, y2, z2
+	.
+	.
+	.
+	122, x122, y122, z122
+	*ELEMENT, type=CPS4, ELSET=Surface1
+	1, 1, 2, 9, 8
+	2, 2, 3, 10, 9
+	.
+	.
+	.
+	103, 7, 122, 87, 8
+
+Note that the first column in the node or element list is used for indexing purposes. The four other indices in each line of the *ELEMENT list
+are the corner IDs that dictate the element connectivity. These corner IDs are listed to guarantee right-handed element just as with the ISM format.
+The Abaqus element type "CPS4" corresponds to a quadrilateral element. For the three-dimensional variant of this mesh file output we use
+
+	*ELEMENT, type=C3D8, ELSET=Volume1
+
+where "C3D8" corresponds to a hexahedron.
+
+The standard Abaqus file format can be used to create a straight-sided mesh. The high-order boundary information and curvature
+generated by HOHQMesh is output in the second portion of this mesh file. We demarcate between the two portions of the mesh file with the comment
+line
+
+	** ***** HOHQMesh boundary information ***** **
+
+In the Abaqus format anything prefaced with `** ` is treated as a comment and is ignored by an ABAQUS file parser. Therefore, all the HOHQMesh
+information output in the second portion of the mesh file is prefaced with to act as an Abaqus comment. After the above comment line the mesh
+file gives the polynomial degree of the boundary curves in the mesh. Then, this mesh file format closely resembles the ISM format albeit slightly reordered. First, there is a list of the element connectivity, curved edge checks, and polynomial interpolant information after which comes the list of boundary names.
+
+The order of the boundary names is a final difference in the mesh format. The Abaqus format and libraries that use it (e.g. [`p4est`](https://github.com/cburstedde/p4est)) require the boundary labels in a particular order. In general, we can think of a quadrilateral element to have sides labeled
```

Comment:
Is this sentence technically correct? I wasn't aware that p4est cares about boundary labels (and it shouldn't, since it doesn't understand our custom format anyways, does it?). If it's really just Trixi that requires it this way because we follow the convention stipulated by p4est, I suggest to put it this way.

Reply:
p4est cares about boundary labels and ordering (https://p4est.github.io/api/structp4est__connectivity__t.html). The way that the Trixi+p4est implementation goes through the tree and counts how many faces are on the boundary uses a small wrapper function `count_required_surfaces` that effectively calls the p4est library function `p4est_iter_face_info_t` (or `p8est_iter_face_info_t`). So we need to respect the boundary ordering expected by p4est because that is how we walk through the tree and find this information

Reply:
OK, thanks for letting me know!